.env

```diff
@@ -0,0 +1 @@
+DEFAULT_NUMBER_OF_RECORDS=10
```

Comment:
We need to rename this file to `.env.development`. Let's also add empty values for `SENTRY_DSN` and `NEW_RELIC_LICENSE_KEY`